### ISSUE TITLE
KeyError: 'audioModes'

### ISSUE THREAD

**Issue by markus681** [yarrm80s/orpheusdl#8]
Hello,

having this Error when downloading an Artist URL, more info down here....
Using Linux Ubuntu Server 20.04.3, Python 3.8.10

trying to download this url "https://listen.tidal.com/artist/4515808"

```
        Album 5/51
DEBUG:urllib3.connectionpool:https://api.tidal.com:443 "GET /v1/albums/96726215/items/credits?replace=True&offset=0&limit=100&includeContributors=True&countryCode=DE HTTP/1.1" 200 None
Traceback (most recent call last):
  File "orpheus.py", line 223, in <module>
    main()
  File "orpheus.py", line 218, in main
    orpheus_core_download(orpheus, media_to_download, tpm, sdm, path)
  File "/home/markus/orpheusdl/orpheus/core.py", line 386, in orpheus_core_download
    downloader.download_artist(media_id, extra_kwargs=media.extra_kwargs)
  File "/home/markus/orpheusdl/orpheus/music_downloader.py", line 183, in download_artist
    tracks_downloaded += self.download_album(album_id, artist_name=artist_name, path=artist_path, indent_level=2, extra_kwargs=artist_info.album_extra_kwargs)
  File "/home/markus/orpheusdl/orpheus/music_downloader.py", line 107, in download_album
    album_info: AlbumInfo = self.service.get_album_info(album_id, **extra_kwargs)
  File "/home/markus/orpheusdl/modules/tidal/interface.py", line 280, in get_album_info
    if album_data['audioModes'] == ['DOLBY_ATMOS']:
KeyError: 'audioModes'
```

i have checked the Album on the URL provided, if i do the download with the Album URL no Error...

my Settings

```
{
    "global": {
        "general": {
            "download_path": "./downloads/",
            "download_quality": "hifi",
            "search_limit": 10
        },
        "artist_downloading": {
            "return_credited_albums": true,
            "separate_tracks_skip_downloaded": true
        },
        "formatting": {
            "album_format": "{artist} - {name}{explicit} {quality}",
            "playlist_format": "{name}{explicit}",
            "track_filename_format": "{track_number} - {album_artist} - {name}",
            "single_full_path_format": "{name}",
            "enable_zfill": true
        },
        "codecs": {
            "proprietary_codecs": true,
            "spatial_codecs": true
        },
        "module_defaults": {
            "lyrics": false,
            "covers": "default",
            "credits": false
        },
        "lyrics": {
            "embed_lyrics": false,
            "save_synced_lyrics": false
        },
        "covers": {
            "main_compression": "high",
            "main_resolution": 1400,
            "save_external": false,
            "external_format": "png",
            "external_compression": "low",
            "external_resolution": 3000,
            "save_animated_cover": true
        },
        "advanced": {
            "advanced_login_system": false,
            "codec_conversions": {
                "alac": "flac"
            },
            "conversion_flags": {
                "flac": {
                    "compression_level": "0"
                }
            },
            "conversion_keep_original": true,
            "cover_variance_threshold": 8,
            "debug_mode": true,
            "disable_subscription_checks": false,
            "enable_undesirable_conversions": false,
            "ignore_existing_files": false,
            "ignore_different_artists": true
        }
    },
```
Is there any option to not do a "codec_conversions" ??? If it is a MQA then it should be an MQA extensension...

Maybe it's a Bug or something else, i don't Know...

Thanks so far

Greetz Markus

**Comment by markus681** [comment]
I also get the same Error on this URL "https://listen.tidal.com/artist/4328573" direct from the beginning

**Comment by Dniel97** [comment]
Looks like you are having an issue with my Tidal module, so please write an issue on my Tidal module [OrpheusDL Tidal Issues](https://github.com/Dniel97/orpheusdl-tidal/issues) page.

That specific error is already fixed in the `develop` branch with commit [ab07292](https://github.com/Dniel97/orpheusdl-tidal/commit/ab07292349acfab23526f04c6fc81a6ba338d4e0). So just switch to the `develop` branch or wait till the branch gets merged into the `master` branch.
